translations/messages.pt.yaml

```diff
@@ -0,0 +1,8 @@
+nav:
+    homepage: 'Página inicial'
+homepage:
+    main_title: 'Afixos Míticas (Em pt-BR)'

```

Comment:
Hm, wondering, why the bracket?
Right now the point is to precise it uses the europe week system (so new week on wednesday)

Reply:
I got so busy and did not see it!

bracket, (Em pt-BR)??

Reply:
Yeah this bracket
In english it says (On EU), because the website is synced with the EU cycle (so it changes to next week on wednesday morning)

Reply:
Oh, BR = Brazilian and pt = Portuguese it means (In Brazilian Portuguese)

Reply:
You should change `main_title: 'Afixos Míticas (Em pt-BR)'` to `main_title: 'Afixos Míticas (Em EU)'` for it to be correct. It needs to mention EU as in Europe, not the language/country.